pom.xml

```diff
@@ -5,11 +5,12 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.15</version>
+        <version>2.33</version>
+        <relativePath />
     </parent>
 
     <properties>
-        <jenkins.version>1.625.3</jenkins.version>
+        <jenkins.version>1.625</jenkins.version>
```

Comment:
What is the reason for this change?

Reply:
checking if it can be reason of failing jobs... but it not so I will remove it...